api/groups.go

```diff
@@ -156,6 +159,8 @@ func (f *RemoteGroupFetcher) FetchPRRApprovers() ([]string, error) {
 
 	var result []string
 	result = append(result, config.Data[f.PRRApproversAlias]...)
+	// TODO: Figre out if we want to treat emeritus approvers differently.
+	result = append(result, config.Data[f.PRRApproversEmeritusAlias]...)
```

Comment:
Actually - if it's impossible to merge manually, then what we can do is to:
- append explicitly "ehashman" here
- get rid of it in a ubsequent PR 

Let me do that now maybe.

Reply:
thanks!